metricbeat/module/system/process/process.go

```diff
@@ -143,6 +143,32 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) error {
 			rootFields.Put("process.command_line", v)
 		}
 
+		// Duplicate system.process.state with process.state
+		if v, ok := proc["state"]; ok {
+			rootFields.Put("process.state", v)
+		}
+
+		// Duplicate system.process.cpu.start_time with process.cpu.start_time
+		// Duplicate system.process.cpu.total.norm.pct with process.cpu.pct
+		if k, ok := proc["cpu"]; ok {
+			cpu := k.(common.MapStr)
+			if v, ok := cpu["start_time"]; ok {
```

Comment:
I wonder if it makes sense to make a `getAndCopy` function here like `getAndRemove` to cut down on some of this `if`-block cruft, potentially using the `CopyFieldsTo` method for MapStr.

Reply:
Perfect! Will do!